daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md

```diff
@@ -275,4 +235,4 @@ main();
 - [Configure a secret store]({{<ref setup-secret-store>}})
 - [Supported secrets]({{<ref supported-secret-stores>}})
 - [Using secrets in components]({{<ref component-secrets>}})
-- [Secret stores quickstart](https://github.com/dapr/quickstarts/tree/master/tutorials/secretstore)
+- [Secret stores tutorial](https://github.com/dapr/quickstarts/tree/master/tutorials/secretstore)
```

Comment:
We should also add the "new" secrets quickstart secrets to the secrets overview article.